### PR TITLE
Added phone icon to tilt-images

### PR DESCRIPTION
**Pull request by vig9295**
Added the phone icon!

### CODE REVIEW

**Comment by ethaMont** [comment]
hmm can you change the major version to a minor version instead?

File: app/components/Phone.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  displayName: 'Phone',
+
+  render() {
+    const classes = classnames('tilt-icon tilt-icon-phone', this.props.className);
+
+    return (
+      <svg className={classes} viewBox="0 0 14 24">
+          <title>Phone</title>
+          <g className="tilt-icon-main" transform="translate(-97.000000, -612.000000)">
+            <path d="M97,614.004947 C97,612.897645 97.8970601,612 99.0058587,612 L108.994141,612 C110.101946,612 111,612.897026 111,614.004947 L111,633.995053 C111,635.102355 110.10294,636 108.994141,636 L99.0058587,636 C97.8980535,636 97,635.102974 97,633.995053 L97,614.004947 Z M100,614 L108,614 L108,615 L100,615 L100,614 Z M98,616 L110,616 L110,632 L98,632 L98,616 Z M104,635 C104.552285,635 105,634.552285 105,634 C105,633.447715 104.552285,633 104,633 C103.447715,633 103,633.447715 103,634 C103,634.552285 103.447715,635 104,635 Z" id="phone"></path>
```
Comment (BrettBukowski) [inline app/components/Phone.js]: remove the `id` attribute

File: app/components/Phone.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
+const React = require('react');
+const classnames = require('classnames');
+
+const Audience = React.createClass({
```
Comment (BrettBukowski) [inline app/components/Phone.js]: should be named `Phone`

Comment (BrettBukowski) [inline app/components/Phone.js]: ...and actually, use the newer `defineSvg` construct that does away with some of the boilerplate.

e.g. https://github.com/Crowdtilt/tilt-images/blob/master/app/components/Envelope.js

**Comment by vig9295** [comment]
Fixed everything from review!

**Comment by BrettBukowski** [comment]
@ethaMont, @tildedave we went from minor version bumps for adding new icons to patch-level version bumps... <https://github.com/Crowdtilt/tilt-images/releases>. Thoughts on going back to minor version bumps or sticking with patch-level?

**Comment by ethaMont** [comment]
@BrettBukowski let's stay with minor version, my bad for the patch-level

**Comment by BrettBukowski** [comment]
Cool. @vig9295 would you be willing to change the version to 6.10.0?

**Comment by vig9295** [comment]
Should be good to go. Waiting on review!

**Comment by BrettBukowski** [comment]
nice - lgtm

**Comment by ethaMont** [comment]
+1

**Pull request merged**